packages/library/src/send-transaction.ts

```diff
@@ -141,7 +143,8 @@ export function createDefaultTransactionSender({
     };
 }
 
-export async function sendAndConfirmDurableNonceTransaction({
+// TODO: Move into separate package, remove `INTERNAL` marker, and export.
+export async function sendAndConfirmDurableNonceTransaction_INTERNAL({
```

Comment:
Ignore these renames. When these get evicted from `@solana/web3.js` in #2122 they will lose the `INTERNAL` suffix.